internal/utils/mounts.go

```diff
@@ -262,3 +262,18 @@ func GetOemLabel() string {
 	}
 	return runtime.OEM.FilesystemLabel
 }
+
+func ActivateLVM() error {
+	// Remove the /etc/lvm/lvm.conf file
+	// Otherwise, it has a default locking setting which activates the volumes on readonly
+	// This would be the same as passing rd.lvm.conf=0 in cmdline but rather do it here than add an extra option to cmdline
+	_, _ = CommandWithPath("rm /etc/lvm/lvm.conf")
```

Comment:
Maybe we should check for the error here, no?

Reply:
we dont really care to be honest, we are trying to avoid lvm reading that file to not apply the locking and make the volumes read only. Either we delete it or it doesnt exist.